### DISCUSSION TITLE
Review Plugin for the Community

### DISCUSSION THREAD

**Discussion by DevByDoll** [bootscore/bootscore#771] [Show and tell]
Hey Guys, I wrote a plugin that works with your nice Theme... If anywone need it, here is my [Repositories](https://github.com/DevByDoll/Google-Review-Wordpress.git)

It is a google review Plugin for wordpress and bootstrap.

**Comment by crftwrk** [comment]
Cool! Do you have an English version? Because mote people doesn't understand English ;-)

**Reply by DevByDoll** [reply]
Hey I can translate it :) 
Comming soon
